_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+authors: [Kris_Jordens]
+title: 'Introduction to API Management'
+image: /img/kris_Jordens.jpg
+tags: [API Management, API Gateway, Products, Responsibilities]
+category: API
+comments: true
+---
+
+        
+# Introduction to API Management
+
+
+## Table of contents
+* [Introduction to API Management](#Introduction-to-API-Management)
+    * [Problem description](#Problem-description)
+    * [Investigation](#Investigation)
+        * [API Management or API Gateway?](#API-Management-or-API-Gateway)
+        * [API landscape without an API management tool](#API-landscape-without-an-API-management-tool)
+        * [API landscape with an API management tool](#API-landscape-with-an-API-management-tool)
+        * [Responsibilities of an API Management tool](#Responsibilities-of-an-API-Management-tool)
+        * [Challenges](#Challenges)
+            * [Multiple API gateways configured](#Multiple-API-gateways-configured)
+            * [Clustering gateways](#Clustering-gateways)            
+    * [API management products](#API-management-products) 
+    * [Hands-on](#Hands-on)
+        * [Installation](#Installation)
+        * [Setup](#Setup)
+        * [Automation](#Automation)
+    * [Conclusion](#Conclusion)
+
+
+## Problem description
+
+In our current digital world where everyone is creating API’s for exposing data internally and for external partners, we should look on how we can manage and control them.
+Most of us also know that we not only have to build the API, but we need to foresee additional responsibilities with it. Some that are just a necessary evil. Like for example Authorization, Authentication, or certain routing policies, etc.  
+Keeping an overview on all available API's also becomes a struggle these days.
+        
+Well, instead of inventing the wheel over and over again, I searched for the best solution and found that an API Management tool can solve a lot of those problems and even add extra value within your landscape.
+This tool helps you in managing you APIs, setting the right responsibilities on the right level. 
+You can add plenty additional functionalities, but most of all this happens in one tool!
+That’s one of the reasons it is important for your business to implement such a tool in your organization.
+        
+                
+## Investigation
+
+### API Management or API Gateway?
+API management and API gateway are used interchangeably.
+But is it now an API manager, API management tool, or API gateway?
+Let’s dive into the naming’s.  
+An API management tool refers to the application or overall solution of managing APIs.
+This mostly indicates that they are talking about the application and its features.
+A management tool can include one or multiple API Gateways, depending on the solution needed in the organization.
+An API gateway is actually a middleware that is placed in front of your services an act as a central point to administrate, route and secure your services.

```

Comment:
@pvanhees The changes are adapted.
on your question "is middleware the correct terminology here? " 
I understand it's debatable. 
The gateway controls the exchange of information between the client software and the software that manages the corporate data".
It can change the format of the data (for example changing XML's to JSON and visa versa) to enable communication between (legacy) systems.
Those are a few reasons why I think we can call it middleware